### DISCUSSION TITLE
[Clarification] deleteListingVideo not really deleting, but changing video_state

### DISCUSSION THREAD

**Discussion by baahar** [etsy/open-api#820] [Q&A]
I observed that deleting a video does not really delete the video for a listing. It is still returned when getListingVideos is called, only the video_state attribute is "inactive". 

Is this the intended behavior?

**Comment by chasemorell** [comment]
I have the same issue. I believe this is a bug. 

deleteListingVideo needs to actually delete the video. 

I am running into issues where I get an error saying I have reached the maximum number of videos that can be associated with a listing even after "deleting" all the videos. This is very frustrating and prevents any future updates to the listing's video. 

This should be categorized as a bug.

**Reply by yovasx2** [reply]
https://github.com/etsy/open-api/discussions/682#discussioncomment-4357236

**Comment by emergingdzns** [comment]
I know Etsy has a "rate limit" so to speak on video uploading. I think it only allows a certain number of videos to be uploaded per hour or per day for any given listing/user. You will have to ask them about this as I don't have the exact details. I just remember it had come up before. It may or may not have anything to do with your issue, but something to keep in mind.

**Reply by yovasx2** [reply]
https://github.com/etsy/open-api/discussions/682#discussioncomment-4357236

**Comment by yovasx2** [comment]
I have same issue, i got a random limit in videos, and after deleting all of them I got 0 videos in GET /videos but then the POST /videos is getting "maximum limit reached"

**Reply by yovasx2** [reply]
in my case it was a rate limit: https://github.com/etsy/open-api/discussions/682#discussioncomment-4357236
